nav2_costmap_2d/src/clear_costmap_service.cpp

```diff
@@ -59,6 +65,17 @@ void ClearCostmapService::clearExceptRegionCallback(
   clearExceptRegion(request->reset_distance);
 }
 
+void ClearCostmapService::clearAroundCallback(
```

Comment:
I recommend not trying to shorten this. Just call it clearAroundRobotCallback.

Oh, by the way, the coding standard is to use UpperCamelCase or snake_case for method names. I found that out when I was addressing your review comment in anothe PR. I'd leave it for now since the other code is written this way, but for new code, we should get the convention right.